### ISSUE TITLE
Installation failures (using Yarn)

### ISSUE THREAD

**Issue by rigsbyt** [aletheia/iam-policy-generator#26]
Having an issue installing that I think has to do with the usage of NPM (I am using Yarn). This could be a strawman though.

`yarn add iam-policy-generator`
```
error <my_package>/node_modules/iam-policy-generator: Command failed.
Exit code: 127
Command: npm run compile && node ./lib/generator/PolicyGenerator.js; tsc ./lib/generated/*.ts
Arguments: 
Directory:<my_package>/node_modules/iam-policy-generator
Output:
> iam-policy-generator@1.1.1 compile <my_package>/node_modules/iam-policy-generator
> tsc

info: Fetching IAM policy metadata from https://awspolicygen.s3.amazonaws.com/js/policies.js 
info: Saving policy file.
info: Generating TS file containing Supported IAM Services enum.
info: Generating TS file containing AWS Service Policies enums.
info: Generating TS file containing ServiceArn
info: library data built. Please import package and have fun!
```

Actually, it installs everything into node_modules and I can use it, but the error causes it to not persist into my package.json and would cause issues with my CI/CD.

**Comment by aletheia** [comment]
It is a bug we need to fix

**Comment by aletheia** [comment]
Looks like it's related to Typescript not being available on your machine (it's not your fault, since it is not a requirement). Installing `tsc` with `npm I -g typescript` should solve the problem. However, version 2.0 is going to include #19 that removes `tsc` dependency.

**Comment by aletheia** [comment]
Is going to be solved by #19
